src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster.component.ts

```diff
@@ -421,7 +440,7 @@ export class MultiClusterComponent implements OnInit, OnDestroy {
     for (let i = 0; i < count; i++) {
       let label = '';
       if (query[i]) {
-        label = query[i]?.metric?.cluster;
+        label = this.updateClusterName(query[i]?.metric?.cluster);
```

Comment:
why do you need it here?

Reply:
This is to show the formatted name in the utilization chart's labels